src/agentscope/agents/__init__.py

```diff
@@ -9,6 +9,11 @@
 from .rpc_agent import RpcAgentServerLauncher
 from .react_agent import ReActAgent
 
+try:
```

Comment:
1. Why we use "try except" here?
2. Users can see two `None` classes named `RAGAgentBase` and `LlamaIndexAgent` under `agents` module.

Reply:
Moved to example for now as discussed, so the problem does not exist now.